cmd/livepeer_cli/wizard_stats.go

```diff
@@ -76,9 +75,13 @@ func (w *wizard) stats(showOrchestrator bool) {
 		w.delegatorStats()
 	}
 
-	currentRound := w.currentRound()
+	currentRound, err := w.currentRound()
+	if err != nil {
+		glog.Errorf("Error getting current round: %v", err)
+		return
+	}
 
-	fmt.Printf("CURRENT ROUND: %v\n", currentRound)
+	fmt.Printf("CURRENT ROUND: %v\n", currentRound.Int64())
```

Comment:
Don't think we need to convert to an `int64` here.

Reply:
fixed in 7dc95c3